src/services/package-manager/package-service.ts

```diff
@@ -0,0 +1,95 @@
+import {logger} from "../../util/logger";
+import {ContentNodeTransport, PackageDependencyTransport, packageManagerApi} from "../../api/package-manager-api";
+import {v4 as uuidv4} from "uuid";
+import {FileService, fileService} from "../file-service";
+import {BatchExportNodeTransport} from "../../interfaces/batch-export-node-transport";
+import {variableService} from "./variable-service";
+
+class PackageService {
+    protected readonly fileDownloadedMessage = "File downloaded successfully. New filename: ";
+
+    public async listPackages(): Promise<void> {
+        const nodes = await this.findAllPackages();
+        nodes.forEach(node => {
+            logger.info(`${node.name} - Key: "${node.key}"`);
+        });
+    }
+
+    public findAllPackages(): Promise<ContentNodeTransport[]> {
```

Comment:
I think this is not used anywhere outside this class, but also can be directly used as `packageManagerApi.findAllPackages()` where needed.